osu.Game.Tournament/Components/SongBar.cs

```diff
@@ -101,11 +102,25 @@ private void load(OsuColour colours)
 
         private void refreshContent()
         {
-            if (beatmap == null)
+            beatmap ??= new BeatmapInfo
```

Comment:
Is there any significance to this change?

Reply:
Just felt it was weird to display nothing (like the bar disappears) when no beatmap loaded.

Reply:
Well sure but it also means that if the local cutting edge instance is opened and you select a _non-online_ beatmap it'll also show that placeholder.

![image](https://github.com/ppy/osu/assets/20418176/2f42b30e-3f6d-4cf6-9db1-e79e2f90e74e)

For a client that is to be used for broadcast purposes I feel like showing nothing is better than showing that.

Reply:
mmm, i'm willing to go out on a limb on this one and revert if requested by tournament hosts.

but can also revert if you think it's best. just weird when testing the tournament client and a whole element is just missing.